src/lib/caching/structures/channels/NewsChannel.ts

```diff
@@ -16,6 +16,21 @@ export class NewsChannel extends GuildTextChannel {
 	 */
 	public readonly type = ChannelType.GuildAnnouncement;
 
+	/**
+	 * Follows this channel.
+	 * @param webhookChannel
+	 * @since 0.0.1
+	 * @see https://discord.com/developers/docs/resources/channel#channel-object-channel-types
+	 */
+	public async follow(webhookChannel: GuildTextChannel): Promise<APIChannelFollowResult> {
+		const reqData = {
```

Comment:
You may as well just do this in the method below, just to clean it up

Reply:
If you mean like this `return this.client.api.post(Routes.followChannel(this.id), {data: {webhook_channel_id: webhookChannel.id}})`

Reply:
Yes like that but with spaces